code/src/HappyForm.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+
+
+const HappyForm = ({ onMessageChange }) => {
+    const [message , setMessage] = useState ('');
+
+    const MESSAGES_URL ="https://happy-thoughts-technigo.herokuapp.com/thoughts";
+
+    const handleFormSubmit = (event) => {
+    event.preventDefault(); 
+    onMessageChange(setMessage);
+    console.log(JSON.stringify({text: message})) 
+
+     fetch(MESSAGES_URL, {
+        method: "POST",
+        headers:
+        {"Content-Type":"application/json"},
+        body:JSON.stringify({message: message})
+        }
+        ).then(() => {
+            window.location.reload();
+        })
+        .catch(err => console.log("error:", err))
+        
+    };
+
+    return (
+    <>
+        <form onSubmit = {handleFormSubmit}>
+            <section className ="form-conatiner">
```

Comment:
I noticed you have a typo in "form-container" maybe that will bring you future unnecessary frustrations, so keep that in mind ;p